src/System/Random/Internal.hs

```diff
@@ -415,8 +437,7 @@ runStateGenST g action = runST $ runStateGenT g action
 
 -- | The standard pseudo-random number generator.
 newtype StdGen = StdGen { unStdGen :: SM.SMGen }
-  deriving newtype (RandomGen, NFData)
-  deriving stock   (Show)
+  deriving (Show, RandomGen, NFData)
```

Comment:
Haha, a battle of backwards vs. forwards compatibility. Fine, my understanding is that right now `DerivingStrategies` is more of a nice to have than a necessity.

Reply:
Yep, precisely. :smile: